simespresso/io_QE/espresso_class.py

```diff
@@ -0,0 +1,787 @@
+from enum import Enum
+from simphony.core.cuba import CUBA
+from simphony.core.data_container import DataContainer
+from simphony.cuds.particles import Particle, Particles
+from simphony.cuds.lattice import Lattice
+import numpy as np
+import subprocess
+#from simphony.cuds.abstractparticles import ABCParticles
+
+import uuid
+import logging
+
+import os.path
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+
+class qe_functions(object):
```

Comment:
This should be renamed as "Class names should normally use the CapWords convention. " (https://www.python.org/dev/peps/pep-0008/#class-names).

I can't think of any good name suggestions but `QeFuntions` wouldn't be good either as this class contains state of QE and allows one to write the state to QE input files..or read the state from QE output files..